port/unix/omrsysinfo.c

```diff
@@ -1099,13 +1099,20 @@ omrsysinfo_get_number_CPUs_by_type(struct OMRPortLibrary *portLibrary, uintptr_t
 		break;
 
 	case OMRPORT_CPU_TARGET: {
+#if defined(J9OS_I5)
+		toReturn = _system_configuration.ncpus;
```

Comment:
As per @0xdaryl comment could this change be pulled out into a separate commit.  It is fine to be in the same Pull Request